From 72dc450f0e750487915bf85f1e158795b632a593 Mon Sep 17 00:00:00 2001
From: Ankit Agarwal <ankit.agarwal@intel.com>
Date: Fri, 21 Jun 2024 14:30:07 +0530
Subject: [PATCH] Fixed graphics allocator hal issue.

Use aidl instead of hidl for graphics allocator.

Tests: Prepared EB and device is booting fine.

Tracked-On: NA
Change-Id: Ide091c5aaed6d23d2f2be4bcb809ba5c1f416bb6
Signed-off-by: Ankit Agarwal <ankit.agarwal@intel.com>
---
 framework_manifest.xml | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/framework_manifest.xml b/framework_manifest.xml
index 7f08d0c..8bc5118 100644
--- a/framework_manifest.xml
+++ b/framework_manifest.xml
@@ -81,10 +81,9 @@
         </interface>
     </hal>
 
-    <hal format="hidl">
+    <hal format="aidl">
         <name>android.hardware.graphics.allocator</name>
-        <transport>hwbinder</transport>
-        <version>4.0</version>
+        <version>2</version>
         <interface>
             <name>IAllocator</name>
             <instance>default</instance>
-- 
2.34.1

